tests/tests_manifests/test_input_manifest.py

```diff
@@ -256,9 +267,50 @@ def test_neq_update(self) -> None:
         })
         self.assertNotEqual(manifest1, manifest2)
 
-    def test_to_file_formatted(self) -> None:
+    def test_to_file_formatted_schema_version_1_2(self) -> None:
         data_path = os.path.join(os.path.dirname(__file__), "data")
         manifest = InputManifest({
+            "schema-version": "1.2",
+            "build": {
+                "name": "OpenSearch",
+                "version": "3.1.0"
+            },
+            "ci": {
+                "image": {
+                    "linux": {
+                        "tar": {
```

Comment:
can you add a test with all the possible keys?

Reply:
Hi, @rishabh6788 this is a dynamic key means `linux` and `tar` can be `windows` and `zip` and more.

So by testing `linux` and `tar` we already test the possible combination.

You dont have to add all keys because they are not hardcoded to start with. Thanks.